misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/util/DiffUtils.java

```diff
@@ -364,4 +371,19 @@ private static String getModuleConstantName(ModuleConstantDiff moduleConstantDif
             return "unknown";
         }
     }
+
+    /**
+     * Retrieves name of the given {@link ClassDiff} instance.
+     *
+     * @param classDiff class diff instance
+     */
+    private static String getModuleClassName(ClassDiff classDiff) {
+        if (classDiff.getNewNode().isPresent()) {
+            return SyntaxTreeUtils.getClassIdentifier(classDiff.getNewNode().get());
+        } else if (classDiff.getOldNode().isPresent()) {
+            return SyntaxTreeUtils.getClassIdentifier(classDiff.getOldNode().get());
+        } else {
+            return "unknown";
```

Comment:
Shall we move "unknown" to constant for reuse?

Reply:
+1